src/test/scala/org/kibanaLoadTest/simulation/generic/GenericJourney.scala

```diff
@@ -162,7 +179,11 @@ class GenericJourney extends Simulation {
   private val httpProtocol: HttpProtocolBuilder =
     new HttpHelper(config).getProtocol
       .baseUrl(config.baseUrl)
+      // Gatling automatically follow redirects in case of 301, 302, 303, 307 or 308 response status code
+      // Disabling this behavior since we run the defined sequence of requests
       .disableFollowRedirect
+      // Mimic real Chrome browser
+      .maxConnectionsPerHostLikeChrome
```

Comment:
Gatling should allow up to 6 connections per host name by default, but I thought it is better to use Chrome value for visibility. But it basically changes nothing https://github.com/gatling/gatling/blob/045603bc5c9d4466cb97d1e54902451f9b18dadd/gatling-http/src/main/scala/io/gatling/http/protocol/HttpProtocolBuilder.scala#L108

Reply:
That's a good idea but I wonder whether we should use it as it's a deprecated property?

Reply:
Funny I missed that comment. I will just remove it, let's use the default value